### PR TITLE
Service key dynamic

### PR DESCRIPTION
**Pull request by adawalli**
Changed `service_key` to allow dynamic values.

### CODE REVIEW

**Comment by adawalli** [comment]
@yaauie / @colinsurprenant - Let me know if anything else is needed for this PR

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## 3.0.9
+ - Update service key to allow dynamic values.
```
Comment (colinsurprenant) [inline CHANGELOG.md]:
```suggestion
 - Changed `service_key` to allow dynamic values [#16](https://github.com/logstash-plugins/logstash-output-pagerduty/pull/16)
```

**Comment by colinsurprenant** [comment]
Thanks for your contribution @adawalli! 
Changes LGTM - left a minor suggestion for the CHANGELOG.

**Comment by adawalli** [comment]
thanks @colinsurprenant! I applied your suggestion!

**Comment by colinsurprenant** [comment]
Great! :shipit:

**Comment by colinsurprenant** [comment]
v3.0.9 is published! Thanks again @adawalli!

**Comment by adawalli** [comment]
@colinsurprenant - quick question. What does the process look like for this propagating as an installable plugin - (i.e., https://www.elastic.co/guide/en/logstash/7.x/plugins-outputs-pagerduty.html)

**Comment by colinsurprenant** [comment]
@adawalli not sure I get your question?
When I said «v3.0.9 is published» I meant that I pushed the new v3.0.9 plugin gem into rubygems.org https://rubygems.org/gems/logstash-output-pagerduty and at that point v3.0.9 is installable via `bin/logstash-plugin  {install | update} ...` 

I just realized we forgot to update the docs for this change...

**Pull request merged**